pkg/node/ledger/address.go

```diff
@@ -55,17 +55,27 @@ func Addresses(client *node.Client, addr []string) ([]Address, error) {
 	return content, nil
 }
 
-func KeysFiltered(client *node.Client, scAddress string, keyPrefix string) ([]string, error) {
+// keysOfSCFilteredByPrefix returns an array of Key in byte array filtered with a prefix.
+
+// If includePrefix is true, will return all the keys with the given prefix,
+
+// If includePrefix is false, will return all the keys without the given prefix.
+
+//nolint:lll
+func KeysOfSCFilteredByPrefix(client *node.Client, scAddress string, keyPrefix string, includePrefix bool) ([][]byte, error) {
```

Comment:
I think "filterSCKeysByPrefix" would be better as it describes the behavior of the function instead of what values are returned.